docs/blog/crucial-tests/index.md

```diff
@@ -394,6 +392,50 @@ beforeAll(() => {
 });
 ```
 
+## 📦 Test the package as a consumer
+
+a lot of times we don't test our packages as how our consumers will use them,
+this is a simplified example of how we could do that
```

Comment:
Missing here a simple explanation of a problem, e.g. '100% of your library's tests pass, the tests are great, they cover all the scenarios, but shockingly they fail in production, how come?? while the tests work against the local developer files, your user will work against the artifacts that were packed and zipped. If a single file is excluded due to build configuration or becauase of .npmignore, the code will fail...'

I can write myself, but I find it to be a great exercise for Raz the super-technical engineer to explain things in a CTO-level language:)

Reply:
thanks, updated